astroquery/hips2fits/core.py

```diff
@@ -10,6 +10,7 @@
 from astropy import wcs
 
 __all__ = ['hips2fits', 'hips2fitsClass']
+__doctest_skip__ = ['hips2fitsClass.*']
```

Comment:
@tinumide - when we get here, we should also look for doctest skips like this in the code files, and turn on the remote testing for them.

Reply:
Alright, noted